tests/test_statevec_backend.py

```diff
@@ -95,6 +96,7 @@ def test_init_success(self, hadamardpattern, fx_rng: Generator) -> None:
     def test_init_fail(self, hadamardpattern, fx_rng: Generator) -> None:
         rand_angle = fx_rng.random(2) * 2 * np.pi
         rand_plane = fx_rng.choice(np.array([i for i in Plane]), 2)
+        rand_plane = fx_rng.choice(np.array([i for i in Plane]), 2)
```

Comment:
This line is duplicated.

Reply:
Deleted in d3b995f073034a49b5763615417da1f8d6caa59e .